### PR TITLE
Add sign-up API.

### PR DESCRIPTION
**Pull request by cisko3000**
Provides a page with js code to paste into any landing page. This way, a person can create landing pages at different places and hosting services, and collect landing page form submissions. The forms can be custom, and custom fields will be stored as json in a database column.

### CODE REVIEW

**Comment by mjhea0** [comment]
@cisko3000 please resolve the merge conflicts,

You may need to rebase, as I put some changes through last night.

**Comment by cisko3000** [comment]
@mjhea0 Ok, let me know if I did any of this git stuff wrong.

File: app/mod_api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from flask import Blueprint
```
Comment (mjhea0) [inline app/mod_api/__init__.py]: Can you move this to a *views.py* file and use absolute imports?

**Comment by mjhea0** [comment]
I made some changes. Can you address this issue: https://github.com/mjhea0/Flask-Landing/pull/3/files#r265551498? Thanks!

File: app/mod_api/__init__.py (hunk 1 above)
Comment (cisko3000) [reply app/mod_api/__init__.py]: Yeah, but what do you think about calling it endpoints.py or resources.py?

Comment (mjhea0) [reply app/mod_api/__init__.py]: *endpoints.py* is fine

Comment (cisko3000) [reply app/mod_api/__init__.py]: I've made the change

**Comment by mjhea0** [comment]
Merged! Want to add some tests for the new endpoint?

**Comment by cisko3000** [comment]
Sure

**Pull request merged**